hack/templates/values/values.template

```diff
@@ -201,7 +201,7 @@ operator:
                 operator: In
                 values: [ "" ]
   # -- Network Operator image repository.
-  repository: nvcr.io/nvstaging/mellanox
+  repository: {{ .NetworkOperator.Repository }}
   # -- Network Operator image name
   image: network-operator
```

Comment:
```suggestion
  image: {{ .NetworkOperator.Image }}
```

We should change this also back to the values

Reply:
Done